### PR TITLE
[DDW-1129] Stake pool data presentation

### PR DESCRIPTION
**Pull request by daniloprates**
This PR updates the information shown in the Stake Pool tooltip.

## Todos

- [x] Add the "Produced blocks" information
- [x] Format "Produced blocks" and "Controlled stake" to short number
- [x] Remove "Profit margin" 
- [x] Fix colors for "Produced blocks" and "Controlled stake" on all themes

### Text copy
**English:** Produced blocks
**Japanese:** 生成ブロック数

## Screenshots
Screenshot 2019-12-13 at 11 59 37
Screenshot 2019-12-13 at 11 59 45

---

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1576179532393500)
- [ ] Test

---

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Comment by a-rukin** [comment]
image
@daniloprates storybook went crazy :o

**Comment by nikolaglumac** [comment]
@a-rukin I can not reproduce that:

Screenshot 2019-12-13 at 09 48 24

File: source/renderer/app/stores/StakingStore.js [hunk 1]
```diff
@@ -21,6 +21,8 @@ import StakePool from '../domains/StakePool';
 import REWARDS from '../config/stakingRewards.dummy.json';
 
 export default class StakingStore extends Store {
+  STAKE_POOLS_INITIAL_INTERVAL = 1000; // 1 second | unit: 1000 milliseconds
```
Comment (yakovkaravelov) [inline source/renderer/app/stores/StakingStore.js]: Any reason why we have these variables as member variables of `StakingStore` class?
If these are constants then no need to store them as member of class.
We can declare them in constants file, and export for external usage.

File: source/renderer/app/utils/formatters.js [hunk 2]
```diff
@@ -19,7 +13,30 @@ export const formattedWalletAmount = (
...
+    formattedAmount = shortNumber(amount);
+  }
+  const { decimalSeparator } = BigNumber.config().FORMAT;
+  if (!long && decimalSeparator !== '.') {
+    // Only BigNumber.toFormat() method is applying correct separators.
+    // Since this method is not used for condensed format (long = false)
+    // the correct number format has to be applied manually.
+    formattedAmount = formattedAmount.split('.').join(decimalSeparator);
+  }
+  if (withCurrency) formattedAmount += ' ADA';
```
Comment (yakovkaravelov) [inline source/renderer/app/utils/formatters.js]: What about using ES6 syntax?
if (withCurrency) formattedAmount = `${formattedAmount} ADA`;

File: source/renderer/app/utils/formatters.js [hunk 3]
```diff
@@ -19,7 +13,30 @@ export const formattedWalletAmount = (
   let formattedAmount = '';
   if (long) {
```
Comment (DeeJayElly) [inline source/renderer/app/utils/formatters.js]: Please use ternary operator condition

Comment (nikolaglumac) [reply source/renderer/app/utils/formatters.js]: @DeeJayElly can you please fix this one?

File: source/renderer/app/utils/formatters.js (hunk 2 above)
Comment (nikolaglumac) [reply source/renderer/app/utils/formatters.js]: @yakovkaravelov can you please fix this?

File: source/renderer/app/stores/StakingStore.js (hunk 1 above)
Comment (nikolaglumac) [reply source/renderer/app/stores/StakingStore.js]: @yakovkaravelov can you please fix that?

File: source/renderer/app/components/staking/stake-pools/StakePoolTooltip.js [hunk 4]
```diff
@@ -39,16 +39,21 @@ const messages = defineMessages({
     defaultMessage: '!!!Controlled stake:',
     description: '"Controlled stake" for the Stake Pools Tooltip page.',
   },
-  profitMargin: {
-    id: 'staking.stakePools.tooltip.profitMargin',
-    defaultMessage: '!!!Profit margin:',
-    description: '"Profit margin" for the Stake Pools Tooltip page.',
-  },
+  // profitMargin: {
```
[the remark this answers is not in the document]
Comment (nikolaglumac) [reply source/renderer/app/components/staking/stake-pools/StakePoolTooltip.js]: @yakovkaravelov this should NOT have been removed. Please restore this one and make sure the Translations are saved!

File: source/renderer/app/config/stakingStakePools.dummy.json [hunk 5]
```diff
@@ -11,14 +11,19 @@
     "performance": 0,
     "pledge": "68",
     "pledgeAddress": "https://amira.net",
-    "producedBlocks": 82,
+    "producedBlocks": 429856,
+    "metrics": {
```
Comment (nikolaglumac) [inline source/renderer/app/config/stakingStakePools.dummy.json]: @daniloprates I don't think we are using this anywhere? If not we should remove it.

File: source/renderer/app/config/stakingStakePools.dummy.json [hunk 6]
```diff
@@ -11,14 +11,19 @@
...
+      "controlled_stake": 29384700000,
+      "produced_blocks": 429856
+    },
+    "apparent_performance": 92387293,
```
Comment (nikolaglumac) [inline source/renderer/app/config/stakingStakePools.dummy.json]: @daniloprates is this used? If not we should remove it.

File: source/renderer/app/config/stakingStakePools.dummy.json [hunk 7]
```diff
@@ -1,6 +1,6 @@
 [
   {
-    "controlledStake": 7,
+    "controlledStake": 2938470000000,
```
Comment (nikolaglumac) [inline source/renderer/app/config/stakingStakePools.dummy.json]: @daniloprates this is the `controlledStake` shown in Storybook - can you please update `generatStakePoolsFakeData` and generate a new dummy file with higher range of values for `controlledStake` and for `producedBlocks`? We need to test our short numer formatting!

File: source/renderer/app/config/stakingStakePools.dummy.json (hunk 6 above)
Comment (nikolaglumac) [reply source/renderer/app/config/stakingStakePools.dummy.json]: @daniloprates I have removed this part now.

File: source/renderer/app/config/stakingStakePools.dummy.json (hunk 5 above)
Comment (nikolaglumac) [reply source/renderer/app/config/stakingStakePools.dummy.json]: @daniloprates I have removed this part now.

File: source/renderer/app/utils/formatters.js (hunk 3 above)
Comment (DeeJayElly) [reply source/renderer/app/utils/formatters.js]: Done

File: source/renderer/app/config/stakingStakePools.dummy.json (hunk 7 above)
Comment (nikolaglumac) [reply source/renderer/app/config/stakingStakePools.dummy.json]: @yakovkaravelov will take care of this. Thanks!

**Pull request merged**